apps/web/src/components/ExperienceCard/WorkContent/CafContent.tsx

```diff
@@ -0,0 +1,50 @@
+import { useIntl } from "react-intl";
+
+import { commonMessages, getLocalizedName } from "@gc-digital-talent/i18n";
+import { WorkExperience } from "@gc-digital-talent/graphql";
+import { Separator } from "@gc-digital-talent/ui";
+
+import { getExperienceFormLabels } from "~/utils/experienceUtils";
+
+import ContentSection from "../ContentSection";
+import { ContentProps } from "../types";
+
+const CafContent = ({
+  experience: { division, cafEmploymentType, cafRank },
+  headingLevel,
+}: ContentProps<Omit<WorkExperience, "user">>) => {
+  const intl = useIntl();
+  const experienceFormLabels = getExperienceFormLabels(intl);
+
+  return (
+    <>
+      <ContentSection
+        title={experienceFormLabels.team}
+        headingLevel={headingLevel}
+        data-h2-border-right="l-tablet(1px solid gray.lighter)"
+      >
+        {division ?? intl.formatMessage(commonMessages.notAvailable)}
```

Comment:
Division will always be null for CAF so it probably ought to not be mentioned or referenced at all on the card

![image](https://github.com/user-attachments/assets/e1100f93-1240-4278-83a0-27dcb532d774)

Reply:
Removed in [fc42f7a](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12208/commits/fc42f7aae2d4af9aa87c470dc990f58a092b43f5)